nodeup/pkg/model/cloudconfig.go

```diff
@@ -17,15 +17,22 @@ limitations under the License.
 package model
 
 import (
+	"bufio"
 	"fmt"
 	"k8s.io/kops/pkg/apis/kops"
+	"k8s.io/kops/pkg/apis/kops/util"
 	"k8s.io/kops/upup/pkg/fi"
 	"k8s.io/kops/upup/pkg/fi/nodeup/nodetasks"
+	"os"
 	"strings"
 )
 
 const CloudConfigFilePath = "/etc/kubernetes/cloud.config"
 
+// Required for vSphere CloudProvider
+const MinimumVersionForVMUUID = "1.5.3"
+const VM_UUID_FILE_PATH = "/root/vm_uuid"
```

Comment:
Ok. I noticed cloud_init sets this file. /root/vm_uuid is bad location. It should be /etc/vmware/vm_uuid?

I vaguely remember HP folks added this as config option in vSphereCP config file. Shouldn't we be setting and querying vSphere CP config file?